### ISSUE TITLE
PyPI  only has older versions

### ISSUE THREAD

**Issue by jdugan1024** [glemmaPaul/django-taggit-serializer#17]
Hello.  This is a useful bit of code, but it seems that the most recent version isn't available via PyPI. 

This seems related to #4 but doesn't seem to have been completely resolved.

If I do a `pip install django-taggit-serializer` I get version 0.1.1.  

And trying to install 0.1.4 explicitly fails:

```
$ pip install django-taggit-serializer==0.1.4
Collecting django-taggit-serializer==0.1.4
  Could not find a version that satisfies the requirement django-taggit-serializer==0.1.4 (from versions: 0.1.0, 0.1.1)
No matching distribution found for django-taggit-serializer==0.1.4
```

If I  If I go here: https://pypi.python.org/simple/django-taggit-serializer/  I only see:

```
Links for django-taggit-serializer

django-taggit-serializer-0.1.0.tar.gz
django-taggit-serializer-0.1.1.tar.gz
```

Would it be possible to push the latest version to PyPI?

Thanks!

**Comment by jdugan1024** [comment]
BTW, in the meantime this works:

`pip install git+https://github.com/glemmaPaul/django-taggit-serializer.git@0.1.4`

**Comment by glemmaPaul** [comment]
@jdugan1024 Thanks, let me try to get a fix, maybe I need to bump the version to 0.1.5, Pip is having troubles submitting it to 0.1.4 for some reason

**Comment by jdugan1024** [comment]
@glemmaPaul bumping the version would probably be good since there has been at least one commit/fix after you tagged 0.1.4.  (Which it turns out we need, so we're actually using a SHA hash in our requirements.txt to get the very latest code, eg:

```
pip install git+https://github.com/glemmaPaul/django-taggit-serializer.git@38ed1015a698ecc64f512fe640411c829c56847b
```

Thanks for looking into this and thanks for this package!

**Comment by glemmaPaul** [comment]
@jdugan1024 New [version](https://github.com/glemmaPaul/django-taggit-serializer/releases/tag/0.1.5) is up. Thank you for creating the issue, it helped me a lot!
